.github/workflows/dotnet.yml

```
@@ -0,0 +1,93 @@
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+      - 'PRDs/**'
+      - 'LICENSE'
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)